src/Xamarin.Android.Build.Tasks/Utilities/RtxtWriter.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Linq;
+using Microsoft.Android.Build.Tasks;
+using Microsoft.Build.Utilities;
+
+namespace Xamarin.Android.Tasks
+{
+	/// Write a list of Item to a file
+	///
+	public class RtxtWriter {
+		public void Write (string file, IList<R> items)
+		{
+			var sb = new StringBuilder ();
+			foreach (var item in items) {
+				sb.AppendLine (item.ToString ());
+			}
+			Files.CopyIfStringChanged (sb.ToString (), file);
```

Comment:
Can this one be:
```suggestion
			using var sw = MemoryStreamPool.Shared.CreateStreamWriter ();
			foreach (var item in items) {
				sw .WriteLine (item.ToString ());
			}
			Files.CopyIfStreamChanged (sw.BaseStream, file);
```
Then we won't create a potentially large string & reuse a `MemoryStream`.